packages/trip-form/i18n/en-US.yml

```diff
@@ -6,6 +6,30 @@ otpUi:
     depart: Depart at
     now: Leave now
     time: Time
+  ModeSelector:
+    labels:
+      BICYCLE: Bike
+      CAR: Drive
+      RENT: Rent
+      TRANSIT: Transit
```

Comment:
It looks like these might need to be lowercase? 
<img width="369" alt="Screenshot 2023-03-30 at 12 12 34 PM" src="https://user-images.githubusercontent.com/86619099/228899041-a9ead998-ef93-44ee-b6d4-00efa6400478.png">

Reply:
Yes, you're right! Good catch. I made them lowercase and added a .toLowerCase() in the switch statement.